seckey.h

```diff
@@ -80,16 +80,16 @@ class VariableRounds
 	//! \throws InvalidRounds if the number of rounds are invalid
 	inline void ThrowIfInvalidRounds(int rounds, const Algorithm *alg)
 	{
-#if defined(__BORLANDC__)
-		if (rounds < MIN_ROUNDS || rounds > MAX_ROUNDS)
-			throw InvalidRounds(alg ? alg->AlgorithmName() : std::string("VariableRounds"), rounds);
-#elif (M==INT_MAX)	// Coverity and result_independent_of_operands
-		if (rounds < MIN_ROUNDS)
-			throw InvalidRounds(alg ? alg->AlgorithmName() : "VariableRounds", rounds);	
-#else
-		if (rounds < MIN_ROUNDS || rounds > MAX_ROUNDS)
-			throw InvalidRounds(alg ? alg->AlgorithmName() : "VariableRounds", rounds);
-#endif
+		if (M == INT_MAX) // Coverity and result_independent_of_operands
+		{
+			if (rounds < MIN_ROUNDS)
+				throw InvalidRounds(alg ? alg->AlgorithmName() : std::string("VariableRounds"), rounds);
+		}
+		else
+		{
+			if (rounds < MIN_ROUNDS || rounds > MAX_ROUNDS)
+				throw InvalidRounds(alg ? alg->AlgorithmName() : std::string("VariableRounds"), rounds);
+		}
 	}
 

```

Comment:
We are going to need to keep the Borland test `#if defined(__BORLANDC__)`. That guard was added to avoid a compile failure. Also see:

* [Commit 0892ee411b9f3f1d](http://github.com/weidai11/cryptopp/commit/0892ee411b9f3f1d3923ac25a6eb4c3b336c36b2)
* [Commit 602eb5fa0b742b00](http://github.com/weidai11/cryptopp/commit/602eb5fa0b742b00543205a74f748206bf83d40d)

This is one of the joys of a multi-platform library :)